### PR TITLE
Register Latex + DictFiles

### PR DESCRIPTION
**Pull request by rened**

### CODE REVIEW

**Comment by kmsquire** [comment]
Perhaps LaTeX.jl?

File: DictFiles/url [hunk 1]
```diff
@@ -0,0 +1 @@
+ssh://git@github.com/rened/DictFiles.jl.git
```
Comment (mlubin) [inline DictFiles/url]: Why is this URL using the ssh protocol?

Comment (ViralBShah) [inline DictFiles/url]: Yeah - should use the git protocol.

Comment (rened) [inline DictFiles/url]: Thanks, was not aware of this.

**Comment by rened** [comment]
I thought Latex.jl was simpler, but I can change that of course! @ViralBShah @mlubin what do you think?

**Comment by ViralBShah** [comment]
I think `LaTeX` is the standard spelling, and looks more familiar to my eyes than `Latex`. I don't have a strong opinion otherwise.

**Comment by rened** [comment]
Ok, thanks, I renamed it to LaTeX.

**Pull request merged**